CMakeLists.txt

```diff
@@ -232,7 +232,7 @@ configure_file(${CONFIG_HEADER_TEMPLATE}
         ESCAPE_QUOTES)
 
 if (NOT CMAKE_CROSSCOMPILING)
-    if (BUILD_TESTING)
+    if (NOT DEFINED BUILD_TESTING OR BUILD_TESTING)
         include(CTest)
         add_subdirectory(tests)
     endif()
```

Comment:
CTest defines the `BUILD_TESTING` option, should this be:
```
if (NOT CMAKE_CROSSCOMPILING)
    include(CTest)
    if (BUILD_TESTING)
        add_subdirectory(tests)
    endif()
endif()
```
???

Reply:
no it shouldn't, otherwise this PR wouldn't exist

Reply:
https://cmake.org/cmake/help/v3.1/module/CTest.html

Reply:
I don't care what the docs say.... they;re wrong

Reply:
we could move include(CTest) to the top of the project if you'd like